_posts/2019-12-09-november-2019.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: November 2019 Release Notes
+category: releases
+tags: release-notes
+---
+
+## Improvements
+
+* **Rails Upgrade** We upgraded our infrastructure to a new version of Rails.
+* **Elasticsearch Upgrade:** We upgraded our Elasticsearch infrastructure, including custom indices.
```

Comment:
Please make this say "We upgraded the Elasticsearch version for our custom search features: Best Bets, Tweets, Videos, RSS, Health Topics, Federal Register, and Related Searches."

Reply:
Thanks, just put edits in - please take a second look when you have a chance